wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/LocalGitUserResolver.java

```diff
@@ -34,23 +40,29 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(LocalGitUserResolver.class);
 
-    private final PreferenceDao preferenceDao;
+    private final String                 preferencessUrl;
+    private final HttpJsonRequestFactory requestFactory;
 
     @Inject
-    public LocalGitUserResolver(PreferenceDao preferenceDao) {
-        this.preferenceDao = preferenceDao;
+    public LocalGitUserResolver(@Named("che.api") String apiUrl, HttpJsonRequestFactory requestFactory) {
+        this.preferencessUrl = apiUrl + "/preferences";
```

Comment:
Think about adding tests that will ensure that this path really is PreferencesService path
Like this https://github.com/eclipse/che/pull/4386/files#diff-5bb5d6c3bf632ea1c9cce3ed96523af7R71